_includes/tutorials/message-ordering/kafka/code/build.gradle

```diff
@@ -0,0 +1,63 @@
+buildscript {
+    repositories {
+        jcenter()
+    }
+    dependencies {
+        classpath "com.commercehub.gradle.plugin:gradle-avro-plugin:0.21.0"
+        classpath "com.github.jengelman.gradle.plugins:shadow:4.0.2"
+    }
+}
+
+plugins {
+    id "java"
+    id "com.google.cloud.tools.jib" version "1.1.1"
+    id "idea"
+    id "eclipse"
+}
+
+sourceCompatibility = "1.8"
+targetCompatibility = "1.8"
+version = "0.0.1"
+
+repositories {
+    jcenter()
+
+    maven {
+        url "https://packages.confluent.io/maven"
+    }
+}
+
+apply plugin: "com.commercehub.gradle.plugin.avro"
+apply plugin: "com.github.johnrengelman.shadow"
+
+dependencies {
+    implementation "org.apache.avro:avro:1.10.0"
```

Comment:
Is this needed?

Reply:
@mikebin this line is removed in https://github.com/confluentinc/kafka-tutorials/pull/587/commits/12e0b49c39a1035b172e96a1c1835347b5900090